bin/configure-tower-projects.py

```diff
@@ -667,8 +792,9 @@ def create_workspaces(self) -> Dict[str, TowerWorkspace]:
             Dict[str, TowerWorkspace]:
                 Mapping of project names and their corresponding workspaces
         """
-        for name, users in self.list_projects():
-            ws = TowerWorkspace(self, name, users)
+        for name, _ in self.list_projects():
+            teams = self.teamids_per_project[name]
```

Comment:
Nit: It is more apparent after reading `__init__` that on the instantiation of the TowerOrganization class that both `self.create()` and `self.populate()` is called?  I can't really think of a scenario when `create_workspaces` would be called without the instantiation of the class.